docs/server/source/data-models/transaction-model.rst

```diff
@@ -33,12 +33,12 @@ Here's some explanation of the contents of a :ref:`transaction <transaction>`:
 
 - id: The :ref:`id <transaction.id>` of the transaction, and also the database primary key.
 - version: :ref:`Version <transaction.version>` number of the transaction model, so that software can support different transaction models.
-- **fulfillments**: List of fulfillments. Each :ref:`fulfillment <Fulfillment>` contains a pointer to an unspent asset
+- **inputs**: List of inputs. Each :ref:`input <Input>` contains a pointer to an unspent asset
```

Comment:
An asset can have many shares (e.g. an amount of 300 in the initial CREATE transaction), and those shares can, get spread out across many outputs (up to 300 in my example). If you sum the amounts of an asset across all the unspent outputs, it should always equal the initial amount created.

Therefore it doesn't make sense to talk about an "unspent asset" --- it only makes sense to talk about an unspent output.